Assets/HoloToolkit/SharingWithUNET/Scripts/UNetAnchorManager.cs

```diff
@@ -190,7 +190,7 @@ private void Start()
                 networkTransmitter.DataReadyEvent += NetworkTransmitter_DataReadyEvent;
             }
 #else
-            if (Application.isEditor)
+            if (VRDevice.isPresent)
```

Comment:
Hmm, I believe this returns true on HoloLens, which would break this flow.

Reply:
This was also recently updated.